dbhelper.py

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env python3
+# vim: ts=4 sw=4 expandtab
+
+import psycopg2
+from psycopg2.extensions import AsIs
+import os
+import json
+from os.path import isfile, join
+
+
+def run_insert(cur, sql, d, extra_vals = ()):
```

Comment:
Although this use of a default arg is okay because it's a tuple and nonmutable, it still bugs me because of the problems with mutable sequences; if someone ever changes this to a list for some reason things will get odd.  I'd prefer it to be "None", and then have a "if extral_vals is None: extra_vals = ()".  It seems wasteful but it's an antipattern to me.  As it's not an error, I wouldn't require it.  What do you think?

Reply:
I agree, good idea to be careful here; done.